src/domains/ecommerce-app/04_apim_ecommerce_checkout.tf

```
@@ -108,6 +108,15 @@ resource "azurerm_api_management_api_operation_policy" "get_transaction_info" {
   xml_content = file("./api/ecommerce-checkout/v1/_validate_transactions_jwt_token.tpl")
 }
 
+resource "azurerm_api_management_api_V2_operation_policy" "get_transaction_info_v2" {
```

```suggestion
resource "azurerm_api_management_api_operation_policy" "get_transaction_info_v2" {
```

what is this v2 resource name?

---

a find replace error, accepting correction